src/slack/wincent/constants.js

```diff
@@ -35,3 +35,16 @@ export const ORDER_NOTE_ACTIONS = [
   NOTE_NO_ACTION,
   CANCEL_ORDER_ACTION,
 ]
+
+export const DELIVERY_PLACE_ACTIONS = [
+  {name: 'delivery', text: 'Home', type: 'button', value: 'remote'},
+  {name: 'delivery', text: 'Office', type: 'button', value: 'office'},
+  CANCEL_ORDER_ACTION,
+]
```

Comment:
actually, these new lines indicate a bug. in Wincent, we didn't have this question, and we even don't want it. the problem is, Wincent's flow should start directly with the Personal/Company order step. therefore, we can:
- remove these DELIVERY actions
- remove delivery step
- make sure wincent goes to `type` step after `new` - change in `updateOrder` - I'll put a comment there also

Reply:
Sure, I will remove `delivery` and fix it in `updateOrder`.